lib/features/modeling/behavior/LabelBehavior.js

```diff
@@ -86,7 +79,16 @@ export default function LabelBehavior(
   CommandInterceptor.call(this, eventBus);
 
   // update label if name property was updated
-  this.postExecute('element.updateProperties', function(e) {
+  this.postExecute('element.updateProperties', onPropertyUpdate);
```

Comment:
Do we also cover text annotation and groups (category) here? Is that needed?

Reply:
As you can see below, TextAnnotation is still covered. Not sure about the groups. However, I don't want to creep the scope of this PR.